Runtime/Core/ApiGatewayManagement/AmazonGameLiftClientWrapper.cs

```diff
@@ -42,17 +56,143 @@ public async Task<SearchGameSessionsResponse> SearchGameSessions(SearchGameSessi
             return await _amazonGameLiftClient.SearchGameSessionsAsync(request);
         }
 
-        public static IAmazonGameLift Create(string localEndpoint)
+        public async Task<DescribeGameSessionsResponse> DescribeGameSessions(DescribeGameSessionsRequest request)
+        {
+            return await _amazonGameLiftClient.DescribeGameSessionsAsync(request);
+        }
+
+        private IAmazonGameLift CreateGameLiftClient(RetriveAwsCredentialsResponse credentialsResponse)
+        {
+            return new AmazonGameLiftClient(credentialsResponse.AccessKey, credentialsResponse.SecretKey);
+        }
```

Comment:
This should be moved out of the region block above the region block since this is agnostic of both region blocks?

Reply:
As we have moved the client creation out of this method, we no longer need the method entirely.